### ISSUE TITLE
Lime producing two tables when using np.random.seed

### ISSUE THREAD

**Issue by hiyam97** [marcotcr/lime#354]
Hello,

When I use np.random.seed() before explain_instance (in order to have reproducability for my code) I get two tables in show_in_notebook(), and when I remove it, I don't get this.
Thanks in advance

![Screenshot (251)](https://user-images.githubusercontent.com/35295550/60907329-f49e3280-a281-11e9-9ac1-607c9cc3ef99.png)

**Comment by hiyam97** [comment]
Hello, it turns out that I was trying Lime explainer for several models, one in each cell in jupyter notebook. I solved the issue by changing the random seed from one cell to the other as follows:

In one cell:
```
# defining the LIME explainer for this cell 
explainer = lime.lime_tabular.LimeTabularExplainer(X_train, feature_names=columns_names,
                                                   verbose=True, mode='regression')

# specify instance i which we will do explanation on
i = 25
model = Lasso()
model.fit(X_train, y_train)

# random seed. Should be unique for each model
np.random.seed(20)

# show explanation of instance i
exp = explainer.explain_instance(X_test[i], model.predict, num_features=NUM_FEATURES)
exp.show_in_notebook(show_table=True)
```

In the other cell:
```
explainer = lime.lime_tabular.LimeTabularExplainer(X_train, feature_names=columns_names, 
                                                   verbose=True, mode='regression')
i = 25
model = Ridge()
model.fit(X_train, y_train)
np.random.seed(21)
exp = explainer.explain_instance(X_test[i], model.predict, num_features=NUM_FEATURES)
exp.show_in_notebook(show_table=True)
```

That was pretty much it.
